addons/common/functions/fnc_runQTE.sqf

```diff
@@ -0,0 +1,103 @@
+#include "..\script_component.hpp"
+#include "\a3\ui_f\hpp\definedikcodes.inc"
+/*
+ * Author: john681611
+ * Runs a Quick time Event.
+ *
+ * Arguments:
+ * 0: Object QTE is attached to <OBJECT>
+ * 1: remoteExec command to execute <STRING>
+ * 2: remoteExec command parameters <ARRAY>
+ * 3: remoteExec command target <STRING>
+ * 4: QTE Code <ARRAY>
+ * 5: Max interaction distance from attached object <NUMBER> (default: 10) 
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [car, "hint", "Yay", 1, ["↑", "↓", "→", "←"], 15] call ace_common_fnc_runQTE
+ *
+ * Public: Yes
+ */
+
+params ["_veh", "_command", "_command_params", "_command_exec_code", "_keyCode", ["_max_distance", 10]];
+
+GVAR(QTEVeh) = _veh;
+GVAR(QTECommand) = _command;
+GVAR(QTECommandParams) = _command_params;
+GVAR(QTECommandExecCode) = _command_exec_code;
+GVAR(QTEMaxDistance) = _max_distance;
+GVAR(QTECode) = _keyCode;
+GVAR(QTEHistory) = [];
+
+hint format ["%1: \n %2", localize LSTRING(QTEInput), [GVAR(QTECode)] call FUNC(getFormattedQTECode)];
+
+findDisplay 46 displayAddEventHandler ["KeyUp", {
+    params ["_displayOrControl", "_key", "_shift", "_ctrl", "_alt"];
+    private _override = false;
```

Comment:
Remove variable. There is no logic applied to it, it's only set twice - not worth being a variable.
Remove unnecessary variables that stem from `params` as well.